### PR TITLE
Fix coding style issues #899

### PR DESCRIPTION
**Pull request by jongfeli**

### CODE REVIEW

File: tests/phpunit/includes/storage/StoreTest.php [hunk 1]
```diff
@@ -184,7 +184,7 @@ public function testGetPropertiesSpecial() {
 
 		$this->assertInstanceOf( '\SMW\SQLStore\SimpleListLookup', $result );
 		foreach( $result->fetchResultList() as $row ) {
-			$this->assertEquals( 2, sizeof( $row ) );
```
Comment (JeroenDeDauw) [inline tests/phpunit/includes/storage/StoreTest.php]: /me looks at @mwjames :)

**Comment by JeroenDeDauw** [comment]
Thanks!

File: tests/phpunit/includes/storage/StoreTest.php (hunk 1 above)
Comment (mwjames) [inline tests/phpunit/includes/storage/StoreTest.php]: Nop, this is not a test I wrote, it just happen to use the new class.

Comment (JeroenDeDauw) [inline tests/phpunit/includes/storage/StoreTest.php]: /me looks at @nischayn22 :)

Comment (JeroenDeDauw) [inline tests/phpunit/includes/storage/StoreTest.php]: Damn, just realized that it's still silly. Fixed in https://github.com/SemanticMediaWiki/SemanticMediaWiki/pull/944

Comment (nischayn22) [inline tests/phpunit/includes/storage/StoreTest.php]: Don't remember much about this :/ 
Anyway, thanks for fixing it.

**Pull request merged**